packages/nux/src/index.js

```diff
@@ -9,5 +9,5 @@ export { default as DotTip } from './components/dot-tip';
 deprecated( 'wp.nux', {
 	since: '5.4',
 	hint: 'wp.components.Guide can be used to show a user guide.',
-	version: '6.2',
+	version: '6.8',
```

Comment:
@peterwilsoncc When do you think is the right time to remove this one? It is fairly low impact because it's not loaded by default but I also don't really expect much usage at this point.

Reply:
> What does "no-opped" mean? Functions and components that do nothing?

Yes, that's correct. This would need to include accessing the data store too.